src/cmdict/ecdict_connector.py

```diff
@@ -88,6 +94,7 @@ def query(self, word):
             query = "select * from stardict where word = ?"
             cursor = self._conn.cursor()
             cursor.execute(query, (word,))
+            record(word, path_history)
```

Comment:
I think in this file you do not need `path`. You already have a default path in the `record()` so calling it directly would be good enough. Adding this in the connector makes it confusing.

Reply:
In tests, the temporary path has to be passed to `record`, or the test will leave yaml file behind in the default path.

Reply:
You do not write code based on tests.